e2e/volumes.e2e.spec.ts

```diff
@@ -195,4 +195,23 @@ test.describe.serial('Volumes Tests', () => {
       }
     }
   });
+
+  test('should auto-refresh volumes list', async() => {
+    const volumesPage = new VolumesPage(page);
+    const autoRefreshVolumeName = `auto-refresh-test-${ Date.now() }`;
+
+    await volumesPage.waitForTableToLoad();
+
+    await tool('docker', 'volume', 'create', autoRefreshVolumeName);
```

Comment:
Same as above; we should check that the volume doesn't magically appear before we create it.